modules/compiler/test/lit/passes/lower-to-mux-builtins.ll

```diff
@@ -0,0 +1,83 @@
+; Copyright (C) Codeplay Software Limited
+;
+; Licensed under the Apache License, Version 2.0 (the "License") with LLVM
+; Exceptions; you may not use this file except in compliance with the License.
+; You may obtain a copy of the License at
+;
+;     https://github.com/codeplaysoftware/oneapi-construction-kit/blob/main/LICENSE.txt
+;
+; Unless required by applicable law or agreed to in writing, software
+; distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+; WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+; License for the specific language governing permissions and limitations
+; under the License.
+;
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+; RUN: muxc --passes lower-to-mux-builtins,verify -S %s | FileCheck %s
+
+; Check that the replace-wgc correctly defines the work-group collective functions
```

Comment:
If this is calling lower-to-mux-builtins, why does it mention replace-wgc here?

Reply:
copy/paste :)